ybn/operators.py

```diff
@@ -156,18 +161,23 @@ def execute(self, context):
         else:
             parent = None
 
-        if bound_type in [SollumType.BOUND_COMPOSITE, SollumType.BOUND_GEOMETRYBVH]:
-            bound_obj = create_empty_object(bound_type)
-            bound_obj.parent = parent
+        bound_obj = create_empty_object(bound_type)
+        bound_obj.parent = parent
 
-            return {"FINISHED"}
+        # Always apply the selected flag preset for all bound types
+        index = context.window_manager.sz_flag_preset_index
+        load_flag_presets()
 
-        bound_obj = create_bound_shape(bound_type)
```

Comment:
This is broken now. All bounds are created as empties. Empties should only be created for composite and BVH types, the other should use `create_bound_shape` so the correct mesh is applied to them.